### PR TITLE
fix #270

### LINKED ISSUE #270: Move labman tables out of qiita schema and into own new schema
Per @antgonza:

"Control access via schemas in the same db so we prevent any potential "bad" code from labman to jump onto qiita."

"the goal is to split the main interactions between qiita/labman; ... and perhaps going with the idea of separate schemas is the way to go. Separating in schemas might give us some nice things, from the pgsql documentation:
     . To allow many users to use one database without interfering with each other.
     . To organize database objects into logical groups to make them more manageable.
     . Third-party applications can be put into separate schemas so they do not collide with the names of other objects"

### PR DESCRIPTION
**Pull request by antgonza**
There a lot of changes but they are pretty basic. Anyway, suggest 2 reviewers ... once tests pass ...

### CODE REVIEW

File: labman/db/process.py [hunk 1]
```diff
@@ -3038,53 +3038,53 @@ def generate_prep_information(self):
...
-                LEFT JOIN qiita.pool_composition pccon ON (
+                LEFT JOIN labman.pool_composition pccon ON (
                     pcc1.input_composition_id = pccon.composition_id)
-                 LEFT JOIN qiita.pool_composition_components pcc2 ON (
+                 LEFT JOIN labman.pool_composition_components pcc2 ON (
                     pccon.pool_composition_id =
                     pcc2.output_pool_composition_id)
                 -- Retrieve amplicon library prep information
-                LEFT JOIN qiita.library_prep_16S_composition lp ON (
+                LEFT JOIN labman.library_prep_16s_composition lp ON (
```
Comment (wasade) [inline labman/db/process.py]: Capitalization change here

Comment (antgonza) [reply labman/db/process.py]: As you know, tried to fix this: https://github.com/jdereus/labman/issues/279 here but it was getting too confusing due to number of changes and travis errors; thus, decided to open an issue about it.

Comment (wasade) [reply labman/db/process.py]: :+1:

File: labman/db/process.py [hunk 2]
```diff
@@ -3114,56 +3117,57 @@ def generate_prep_information(self):
...
                     normalization_process_id)
                 -- Retrieving compressed gdna information
-                LEFT JOIN qiita.compressed_gdna_composition cgc USING (
+                LEFT JOIN labman.compressed_gdna_composition cgc USING (
                     compressed_gdna_composition_id)
                 -- Retrieving gdna information
-                LEFT JOIN qiita.gdna_composition gc USING (gdna_composition_id)
-                LEFT JOIN qiita.composition c2 ON (
+                LEFT JOIN labman.gdna_composition gc
+                    USING (gdna_composition_id)
```
Comment (AmandaBirmingham) [inline labman/db/process.py]: I see a few places like this where what was one line previously has been broken into two.  I have no problem with it, just curious: what's the rationale?  Is it staying below the flake8-enforced line length, or is it some convention?

Comment (antgonza) [reply labman/db/process.py]: Yup, flake8 line length convention.

**Comment by AmandaBirmingham** [comment]
Ping @wasade : If you approve this PR, could you mark it approved (and I'll merge it)--or just merge it yourself? :) Thanks!

**Pull request merged**